README.md

```diff
@@ -197,11 +232,11 @@ Here's an example for integrating an adapter for [keen.io](https://keen.io)
         writeKey: "ENTER YOUR WRITE KEY"
     });
     var tracking_adapter = {
-        experiment_start: function(experiment_name, variant) {
-            keen_client.addEvent(experiment_name, {variant: variant, event: 'participate'});
+        experiment_start: function(experiment, variant) {
+            keen_client.addEvent(experiment.name, {variant: variant, event: 'participate'});
         },
-        goal_complete: function(experiment_name, variant, event_name) {
-            keen_client.addEvent(experiment_name, {variant: variant, event: event_name});
+        goal_complete: function(experiment, variant, event_name, _props) {
```

Comment:
by convention variables that are passed over but not used are marked with `_` prefix. I think it's useful to know that it's being passed over from the experiment, even if it's not used.